Library/Homebrew/cmd/migrate.rb

```diff
@@ -0,0 +1,29 @@
+require "migrator"
+require "formula_renames"
+
+module Homebrew
+  def migrate
+    raise FormulaUnspecifiedError if ARGV.named.empty?
+
+    ARGV.named.each do |name|
```

Comment:
Oh, I see. I think I have a better way to solve it. You can change the logic inside `Formulary#from_rack` in this way:

```ruby
    if tap.nil?
      factory(rack.basename.to_s, spec)
    elsif  tap == "Homebrew/homebrew"
      factory(Fomulary.core_path(rack.basename.to_s), spec)
    else
      factory("#{tap.sub("homebrew-", "")}/#{rack.basename}", spec)
    end
```